src/form/bf_mx.jl

```diff
@@ -28,7 +37,19 @@ function variable_mc_voltage_prod_hermitian(pm::AbstractUBFModels; n_cond::Int=3
         (Wr,Wi) = variable_mx_hermitian(pm.model, bus_ids, n_cond;
             set_lower_bound_diag_to_zero=true, name="W", prefix="$nw")
     end
-
+    v_start = exp.((im*2*pi/3).*[0; -1; 1]) #TODO this should be made more generic eventually
+    W_start = v_start*v_start'
+    for (id,_) in Wr
+        for i in 1:3
+            for j in 1:i
+                JuMP.set_start_value(Wr[id][i,j], real.(W_start)[i,j])
```

Comment:
Is there a reason to use `Wr[id]` rather than to include it in the loop definition?